perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -312,6 +312,26 @@ sub inspect : Private {
     $c->stash->{categories} = $c->forward('/admin/categories_for_point');
     $c->stash->{report_meta} = { map { $_->{name} => $_ } @{ $c->stash->{problem}->get_extra_fields() } };
 
+    my @priorities = $c->model('DB::ResponsePriority')->for_bodies($problem->bodies_str_ids)->all;
```

Comment:
I think this is causing some two-tier weirdness with the wrong priorities being shown if you change to a category in a different council:
![priorities](https://cloud.githubusercontent.com/assets/4776/21230724/cf58d83a-c2dd-11e6-9e93-b15f88e4f2c3.gif)
Selecting the priorities by the report's location, not its `bodies_str`, would probably fix this.